### PR TITLE
Added my own clean notepad collection!

### PR DESCRIPTION
**Pull request by GamerBoi153**

### CODE REVIEW

**Comment by hellogoose** [comment]
recommendation: remove the "forked to add notepad collection" in readme

**Comment by GamerBoi153** [comment]
sure!

**Comment by GamerBoi153** [comment]
I think I did it because when I try to edit it does not show the forked project thing. Can you check and verify?

File: README.md [hunk 1]
```diff
@@ -1,5 +1,4 @@
-﻿![Desktop Goose Resource Hub!](https://owo.sh/8kxSLJD.png)
-
+﻿![Desktop Goose Resource Hub!](https://owo.sh/8kxSLJD.png) Added new Notepad Pack!
```
Comment (VukAnd) [inline README.md:1]:
```suggestion
﻿![Desktop Goose Resource Hub!](https://owo.sh/8kxSLJD.png)
```

**Comment by GamerBoi153** [comment]
How do I verify your commit/request?

**Pull request merged**